internal/platform/platform_versioner_test.go

```diff
@@ -135,3 +135,90 @@ func TestK8SBasedPlatformVersioner_GetPlatformInfo(t *testing.T) {
 		}
 	}
 }
+
+func TestClientCallVersionComarsion(t *testing.T) {
```

Comment:
small change - typo, Comparison missing the 'p'

Reply:
corrected.